setup.cfg

```diff
@@ -0,0 +1,33 @@
+[metadata]
+name = data_clusterer
+version = attr: data_clusterer.__version__
+description = Application for clustering input data.
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Filip Horák
+license = MIT
+license_files = LICENSE
+platforms = unix, linux, osx, cygwin, win32
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.10
+
+[options]
+packages = data_clusterer
+install_requires =
+    numpy>=2.0.0
```

Comment:
it will be better to add the dependencies as they are needed (not in initial commits)